feature-detects/mediastream.js

```diff
@@ -0,0 +1,21 @@
+/*!
+{
+  "name": "MediaRecorder API",
+  "property": "mediarecorder",
+  "caniuse": "mediarecorder",
+  "tags": ["mediarecorder"],
+  "authors": ["Jaikishan Brijwani"],
+  "notes": [{
+    "name": "MDN Docs",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/API/MediaStream_Recording_API"
+  }]
+}
+!*/
+/* DOC
+Detects support for the MediaStream Recording API for detecting mediastreaming.
+*/
+
+
+define(['Modernizr'], function (Modernizr) {
+    Modernizr.addTest('mediarecorder', typeof MediaRecorder);
+})
```

Comment:
Okay, this took me longer than I though.

1. There is a `;` missing at the end
2. `typeof MediaRecorder` returns a string but the `addTest` needs a `true` or `false` not a string so we have to make a comparison so we get a true or false

The final result should be this:
![Modernizr](https://user-images.githubusercontent.com/45839898/94997322-9bb76a00-05aa-11eb-96e4-a226244d26e2.png)

Reply:
sorry for the mislead there

Reply:
is it ready to be merged now then?